redux/reducers/recipes.ts

```diff
@@ -260,6 +272,17 @@ const recipeSlice = createSlice<
     ids: ["empty_recipe"],
   },
   reducers: {
+    setNutriments: (state, action) => {
+      state.recipes.modifiedRecipe.nutriments = action.payload;
+    },
+    setModifiedRecipe: (
+      state,
+      action: PayloadAction<RecipeStateType>
+    ) => {
+      state.recipes.modifiedRecipe = action.payload;
+   
+    },
```

Comment:
Maybe better to go in `parseURLParameters` and initialise both recipes at the same time.

Still remain the issue of sending a request to the server to the the nutrition information about the initial recipe